controller/companyController.js

```diff
@@ -143,7 +143,8 @@ const getByNameContains = (req, res) => {
     }
 
     // since a user can be only part of one company, only one is being sent back
-    Company.findOne({ name: { $regex: `.*${companyName}.*` }, _id: new Mongoose.Types.ObjectId(user.companyID) }, { _id: 0 }).then(company => {
+    // case insensitive search
+    Company.findOne({ name: { $regex: `.*${companyName}.*`, '$options': 'i' }, _id: new Mongoose.Types.ObjectId(user.companyID) }, { _id: 0 }).then(company => {
```

Comment:
What is the point of this regex if the company must have the user's company ID. What is the use case where this query is used?

Reply:
If we are to make an admin app for our internal use, and management of companies and super-admins this could come in handy. +practice

Reply:
In this phase only basic CRUD operations should be implemented. After that we will discuss about new functionalities. Probably it will come in handy, but should be left for the next phase.

Reply:
Agreed, but since it is there anyhow, I do not see a reason to delete.

Reply:
YOU MUST DELETE IF DRAGAN SAYS SO